doc/source/myrocks/zenfs.rst

```diff
@@ -11,101 +11,190 @@ which places files into zones on a raw zoned block device (ZBD) using
 the MyRocks File System interface. ZenFS depends on ``libzbd`` and requires
 a Linux kernel implementation which supports NVMe Zoned Namespaces. The kernel
 must be configured with `zone block device support
-enabled <https://zonedstorage.io/linux/config/#kernel-configuration>`__.
-
-The ``libzbd`` library provides functions and manages the state of the zones
-of zoned block devices. You can download the
-library `here <https://ubuntu.pkgs.org/21.04/ubuntu-universe-amd64/libzbd1_1.2.0-1_amd64.deb.html>`__ and follow the installation instructions on the Web site. For more information, see `libzbd User Library <https://zonedstorage.io/projects/libzbd/>`__.
+enabled <https://zonedstorage.io/docs/linux/config>`__.
 
 For more information, see `Western Digital and Percona deliver
 Utrastar DC ZN540 Zoned Namespace SSD support for Percona Server for
 MySQL <https://documents.westerndigital.com/content/dam/doc-library/en_us/assets/public/western-digital/collateral/company/western-digital-zns-ssd-perconal-blogpost.pdf>`__.
 
-.. _zenfs-install:
-
-Installing MyRocks with ZenFS packages
-------------------------------------------
-
-The MyRocks with ZenFS packages are listed in
-the :ref:`installing_from_binary_tarball` section.
 
-To install the packages, run the following command:
-   
-.. code-block:: bash
-    
-    $ sudo apt install percona-server-server-zenfs
-    $ sudo apt install percona-server-rocksdb-zenfs
-
-Running the Installation
-----------------------------
-
-The following steps successfully run Percona Server with MySQL on the  ``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>``:
+The following procedure installs and configures Percona Server for MySQL on the  ``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>``.
```

Comment:
I find this sentence somewhat confusing. We're not actually **installing** PS on that device, we rather install using the regular package installation workflow, but then configure it to store it's **data** on ZenFS.

Reply:
Fixed.